### PR TITLE
Feature/1 deployment

### PR DESCRIPTION
**Pull request by he-james**

### CODE REVIEW

**Review by FB-Reda — CHANGES_REQUESTED** [review]
Fix the link

File: pages/index.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ export default function Home() {
 
       <main className={styles.main}>
         <h1 className={styles.title}>
-          Welcome to <a href="https://nextjs.org">Next.js!</a>
+          Welcome to <a href="https://cimun.org">CIMUN XVIII</a>!
```
Comment (sdmungov) [inline pages/index.js]: this is bad

**Review by sdmungov — CHANGES_REQUESTED** [review]

**Review by FB-Reda — APPROVED** [review]

**Review by sdmungov — APPROVED** [review]
Close Enough

**Comment by FB-Reda** [comment]
Lets use this as an example of how we could do better as an organization going forward.

**Pull request merged**